boards/arm/rpi_pico/doc/index.rst

```diff
@@ -112,6 +112,17 @@ Default Zephyr Peripheral Mapping:
 - ADC_CH2 : P28
 - ADC_CH3 : P29
 
+Programmable I/O (PIO)
```

Comment:
Please squash this with the previous commit. This would get rid of that merge artifacts there.